war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -304,15 +304,17 @@ function findAncestorClass(e, cssClass) {
 function findFollowingTR(input, className) {
     // identify the parent TR
     var tr = input;
-    while (tr.tagName != "TR")
+    if(tr.tagName!=undefined){
+      while (tr.tagName != "TR")
         tr = tr.parentNode;
```

Comment:
can the parentNode be undefined as well? I believe no, but just in case

Reply:
Make sense. So, I have added validation even on parentNode.